Content/Documentation/ScriptingAPI-Documentation.md

```diff
@@ -530,6 +576,12 @@ An integer mask that can be used to group entities together for certain operatio
 
 #### TransformComponent
 Describes an orientation in 3D space.
+- Position : Vector XYZ  -- query the position in world space
```

Comment:
The problem here is that GetPosition() returns position in world space (depending on parenting or animation system, or some other system modified its world matrix explicitly), but the TransformComponent::translation_local, etc are in local space. So these properties should be Translation_local, Rotation_local and Scale_local instead.

Reply:
So, what do we do with SetPosition and the like?